packages/legacy/core/App/screens/ProofRequestDetails.tsx

```diff
@@ -30,8 +31,31 @@ interface ProofRequestAttributesCardParams {
   onChangeValue: (schema: string, label: string, name: string, value: string) => void
 }
 
+const formatIfDate = (
```

Comment:
Should this go into a helper lib with a unit test?

Reply:
done